pkg/security/admission/admission.go

```diff
@@ -73,6 +74,13 @@ func (c *constraint) Admit(a admission.Attributes) error {
 		return nil
 	}
 
+	// if this is an update, see if we are only updating the ownerRef.  Garbage collection does this
+	// and we should allow it in general, since you had the power to update and the power to delete.
+	// The worst that happens is that you delete something, but you aren't controlling the privileged object itself
+	if a.GetOldObject() != nil && oadmission.IsOnlyMutatingOwnerRefs(a.GetObject(), a.GetOldObject()) {
```

Comment:
> @deads2k any chance we could check this only once for different objects? I am not optimistic but asking to be sure

I don't understand the question.  I think this only happens once per admission plugin which is restricting updates with special/additional permissions.